azure-pipelines/consistency-report.yml

```diff
@@ -32,10 +32,10 @@ jobs:
 
       - script: |
           python -m pip install --upgrade pip
-          pip install -r ci_scripts/requirements.txt
+          pip install -r ci_scripts/release_scripts/requirements.txt
         displayName: 'Install requirements'
 
-      - script: python ci_scripts/publish_target_validation_report.py
+      - script: python ci_scripts/release_scripts/publish_target_validation_report.py
```

Comment:
- could we take this initiative to move this script to another folder like `validation_scripts` with its requirements.txt?
- Have also a `utils` folder with all shared scripts

Reply:
Yep, that sounds like a great idea 👍